k8s/namespaces/fees-pay/ccpay-dlq-payment-job/prod-00.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: dead-letter-queue-process
+spec:
+  values:
+    job:
+      schedule: "0 0 * * *"
+      environment:
+        REPORT_NAME: dead-letter-queue-process
```

Comment:
this should be set in the base

Reply:
moved into base, removed env and report name as suggested.